src/cpu/x64/jit_uni_batch_normalization.cpp

```
@@ -2241,7 +2241,7 @@ struct driver_t : public c_compatible {
 
             if (C_blks_thr == 0 || N_thr == 0) continue;
 
-            size_t coff_base = global_C_blk_s * simd_w;
+            size_t coff_base = static_cast<size_t>(global_C_blk_s) * simd_w;
```

same problem, function is wrongly designed to return void while `global_C_blk_s` requires exception handling
I think it's better to mark it as intentional and leave comment that it's not expected that `global_C_blk_s` ever becomes negative

---

Alternatively you could silently return when `global_C_blk_s < 0` which will stop kernel from execution instead of getting segfaults. Not sure if silent returns won't make coverity issue again.

---

I prefer to mark it as intentional or false positive. When `global_C_blk_s < 0`, it means `C_blk_s == -1 && C_blk_e == -1 `(from [thread_balance()](https://github.com/oneapi-src/oneDNN/blob/main/src/cpu/x64/jit_uni_batch_normalization.cpp#L2170)), and then `C_blks_thr == 0`. It won't reach the code here.